extensions/amp-analytics/0.1/test/test-events.js

```diff
@@ -1957,33 +1961,53 @@ describes.realWin('Events', {amp: 1}, (env) => {
           readyPromise = Promise.resolve();
           unlisten = env.sandbox.spy();
           unlisten2 = env.sandbox.spy();
+          unlisten3 = env.sandbox.spy();
           config = {};
 
           eventsSpy = env.sandbox.spy(tracker, 'onEvent_');
 
+          target3 = win.document.createElement('div');
```

Comment:
I don't see the need to introduce the `target3` here? Any reason we want a third element.

Reply:
I felt that it was important to test/show that array selector was using querySelectorAll here... but I guess we talked about this before and agreed that this functionality is handled by analytics root and so should be tested there...

Removing now